tools/bazel.rc

```diff
@@ -7,6 +7,8 @@ run --direct_run
 # Adding in c11 compiling, this really should be done in the android toolchains
 build --cxxopt="-std=c++11"
 
+build --copt -DGRPC_BAZEL_BUILD
```

Comment:
I don't know bazel that well, but does this need to be added to every c/cpp file that ever gets built?

Reply:
Right, I think I'd better make it only used for GRPC library